tests/python/driver/tvmc/test_compiler.py

```diff
@@ -56,6 +53,15 @@ def test_compile_tflite_module(tflite_mobilenet_v1_1_quant):
     assert type(dumps) is dict
 
 
+def test_compile_tflite_module(tflite_mobilenet_v1_1_quant):
+    # Check default compilation.
```

Comment:
```suggestion
def test_compile_tflite_module(tflite_mobilenet_v1_1_quant):
    # some CI environments wont offer flute, so skip in case it is not present
     pytest.importorskip("tflite")
```

Not all CI environments will offer `tflite`.